### PR TITLE
Add reducers and actions

### PR DESCRIPTION
**Pull request by fernando-silvabr66**
Add Redux libraries and functions
Define action types for adding and removing a book.
Set the initial state for the bookstore.
Export Action Creators.

### CODE REVIEW

File: src/redux/configStore.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const configReducer = combineReducers({
+  booksReducer,
+});
```
Comment (TedLivist) [inline src/redux/configStore.js]: - [ ] After setting up the categories-related redux actions and reducer, ensure you import the reducer here, just like `booksReducer`.

- [ ] Add the imported categories reducer to the `combineReducers`.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @fernando-silvabr66,

Good job so far!
✔️ Good documentation
✔️ Set up of books redux
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] I would recommend you fix or remove your screenshot to keep your documentation neat.

  fern-scrnsht

- [ ] You are missing the `src/redux/categories/categories` file as presented in the requirements. Kindly add it and set up the categories-related Action Types and Creators and reducer.

  fern-cate

- [ ] _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const initialState = [];
+
+const CHECK_STATUS = 'CHECK_STATUS';
```
Comment (Mihndim2020) [inline src/redux/categories/categories.js]: - [x] Great job defining the `CHECK_STATUS` action type as requested by the previous code reviewer, but according to the project[ requirements.](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md) it is supposed to follow the [ducks](https://github.com/erikras/ducks-modular-redux) pattern, please let's 
 fix that. It should be something like so: `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @fernando-silvabr66, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there

 # Status: Requires Changes ♻️
_Check the comments below under the review._

Please do consider, as requested by the previews reviewer:
- [x] Fixing the link to the image in the `image section` of your `readme file` as the will make your readme file more professional. The link to the image is broken, please kindly fix that. 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/categories/categories.js (hunk 2 above)
Comment (fernando-silvabr66) [reply src/redux/categories/categories.js]: Done!

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @fernando-silvabr66, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**